examples/medplum-healthie-importer/src/healthie/allergy.ts

```diff
@@ -0,0 +1,265 @@
+          status
+          updated_at
+        }
+      }
+    }
+  `;
+
+  const result = await healthie.query<{ user: { allergy_sensitivities: HealthieAllergySensitivity[] } }>(query, {
+    patientId,
+  });
+  return result.user?.allergy_sensitivities ?? [];
+}
+
+/**
+ * Converts a Healthie allergy/sensitivity to a FHIR AllergyIntolerance.
+ * @param allergy - The Healthie allergy/sensitivity object.
+ * @param patientReference - The reference to the patient.
+ * @returns A FHIR AllergyIntolerance resource.
+ */
+export function convertHealthieAllergyToFhir(
+  allergy: HealthieAllergySensitivity,
+  patientReference: Reference<Patient>
+): AllergyIntolerance {
+  const fhirAllergy: AllergyIntolerance = {
+    resourceType: 'AllergyIntolerance',
+    identifier: [{ system: HEALTHIE_ALLERGY_ID_SYSTEM, value: allergy.id }],
+    clinicalStatus: {
+      coding: [
+        {
+          system: 'http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical',
+          code: mapHealthieStatusToClinicalStatus(allergy.status),
+        },
+      ],
+    },
+    verificationStatus: {
+      coding: [
+        {
+          system: 'http://terminology.hl7.org/CodeSystem/allergyintolerance-verification',
+          code: 'confirmed',
+        },
+      ],
+    },
+    type: mapHealthieCategoryToType(allergy.category),
+    category: [mapHealthieCategoryTypeToCategory(allergy.category_type)],
+    criticality: mapHealthieSeverityToCriticality(allergy.severity),
+    patient: patientReference,
+    onsetDateTime: allergy.onset_date?.trim() || undefined,
+    code: {
+      text: allergy.name,
+      coding: [
+        {
+          system: HEALTHIE_ALLERGY_CODE_SYSTEM,
+          code: allergy.name,
+          display: allergy.name,
+        },
+      ],
+    },
+    reaction: allergy.reaction
+      ? [
+          {
+            substance: {
+              text: allergy.name,
+              coding: [
+                {
+                  system: HEALTHIE_ALLERGY_CODE_SYSTEM,
+                  code: allergy.name,
+                  display: allergy.name,
+                },
+              ],
+            },
+            manifestation: [
+              {
+                text: allergy.reaction,
+                coding: [
+                  {
+                    system: HEALTHIE_REACTION_CODE_SYSTEM,
+                    code: allergy.reaction,
+                    display: allergy.reaction,
+                  },
+                ],
+              },
+            ],
+            severity: mapHealthieSeverityToReactionSeverity(allergy.severity),
+          },
+        ]
+      : undefined,
+    note: allergy.reaction ? [{ text: allergy.reaction }] : undefined,
+  };
+
+  return fhirAllergy;
+}
+
+/**
+ * Maps Healthie status values to FHIR clinical status values.
+ * @param status - The status value from Healthie.
+ * @returns A FHIR-compliant clinical status code.
+ */
+export function mapHealthieStatusToClinicalStatus(status?: string): string {
```

Comment:
Is it worth having a more precise return type here? Either something from `@medplum/fhirtypes` or `'active' | 'inactive' | 'resolved'`?